### PR TITLE
Calculate capacity

### PR DESCRIPTION
**Pull request by alaroia-cfa**
Co-authored-by: Asheesh Laroia <alaroia@codeforamerica.org>
Co-authored-by: Jenny Heath <jheath@codeforamerica.org>
Co-authored-by: Em Barnard-Shao <ebarnard@codeforamerica.org>

### CODE REVIEW

**Comment by alaroia-cfa** [comment]
This is just the work @jenny-heath already did on this branch, packaged up as a pull request.

**Comment by alaroia-cfa** [comment]
@embarnard If you're comfortable taking this forward tomorrow, I'd be grateful!

File: app/lib/tax_return_status.rb [hunk 1]
```diff
@@ -45,4 +45,8 @@ def message_templates
   def self.message_template_for(status, locale = "en")
     message_templates[status.to_sym] ? I18n.t(message_templates[status.to_sym], locale: locale) : ""
   end
+
+  def self.statuses_that_count_towards_capacity
```
Comment (bytheway875) [inline app/lib/tax_return_status.rb:49]: In a quick skim, my eye goes straight towards the excepted statuses. Maybe we can set up a const called `STATUSES_EXCLUDED_FROM_CAPACITY = [:intake_before_consent, :intake_in_progress, :file_accepted, :file_not_filing].freeze`, then update this method to use that const. Might be a bit more readable for the people who come after us!

File: app/models/vita_partner.rb [hunk 2]
```diff
@@ -79,28 +76,15 @@ def no_coalitions_for_sites
...
+      errors.add(:capacity_limit, "Sites cannot be assigned a capacity")
+    end
+  end
+
   def one_level_of_depth
     if parent_organization&.parent_organization.present?
       errors.add(:parent_organization, "Only one level of sub-organization depth allowed.")
     end
   end
-
```
Comment (bytheway875) [inline app/models/vita_partner.rb:87]: Woo hoo! Yay for removing old code.

File: app/models/intake.rb [hunk 3]
```diff
@@ -607,4 +603,8 @@ def update_or_create_14446_document(filename)
     )
     document
   end
+
```
Comment (bytheway875) [inline app/models/intake.rb:606]: I'm guessing this is what gets used in the views to display the delay message? Based on the standup convo yesterday, it seems like this works out of the box for us, with no updates needed, right?

**Review by bytheway875 — APPROVED** [review]
👍  One requested change, but not blocking.

**Pull request merged**